src/config/from_document.rs

```diff
@@ -329,3 +336,89 @@ impl TryFrom<ServiceDocument> for Config {
     from_document(value)
   }
 }
+
+#[cfg(test)]
+mod tests {
+
+  use async_graphql::Pos;
+  use async_graphql_value::ConstValue;
+  use indexmap::IndexMap;
+  use serde_json::Number;
+
+  use super::*;
+  use crate::config::Proxy;
+
+  #[test]
+  fn test_upstream_directive_in_server() {
```

Comment:
What are we testing here that's not captured in the other tests?

Reply:
I think the integration test does cover this part.  It is merely a unit test for a certain function.

Reply:
I think we can drop them and just keep the integration tests for this purpose.

Reply:
okay!